addons/api/package.json

```diff
@@ -42,7 +42,7 @@
     "ember-auto-import": "^1.10.1",
     "ember-cli": "~3.27.0",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-inject-live-reload": "^2.1.0",
```

Comment:
FYI I think, before converting this draft to a PR you should perform a rebase with main? I introduced this changes a week ago.

Reply:
you are right. closing this one